streamalert/athena_partition_refresh/main.py

```diff
@@ -55,13 +65,19 @@ def __init__(self):
         config = load_config(include={'lambda.json', 'global.json'})
         prefix = config['global']['account']['prefix']
         athena_config = config['lambda']['athena_partition_refresh_config']
+        store_format = get_data_store_format(config)
+
+        if store_format == 'parquet':
+            self._alerts_regex = self.ALERTS_REGEX_PARQUET
+            self._data_regex = self.DATA_REGEX_PARQUET
+
+        else:
```

Comment:
Maybe 

```
if = 'parquet':
  ...
elif = 'json':
  ..
else:
  Raise
```

Reply:
It is not necessary to have additional check here. I add a new helper to return `None` and trigger `MisConfigurationError` exception if the value neither `parquet` nor `json`.
```
def get_data_store_format(config):
    """Get the data store format using the current config settings
    Args:
        config (CLIConfig): Loaded StreamAlert config
    Returns:
        str: The data store format either "parquet" or "json"
    """
    athena_config = config['lambda'].get('athena_partition_refresh_config', {})

    return athena_config.get('store_format', None)
```